lib/RichTextRenderer.astro

```diff
@@ -19,11 +19,16 @@ export type Props = {
    ```
    */
   resolver: Options["resolver"];
+  stringResolver: Options["stringResolver"];
```

Comment:
Lets give it a thorough description, similar to `resolver` one.

Reply:
done